rosetta/services/transaction_fee.go

```diff
@@ -46,39 +46,39 @@ func computeSuggestedFeeAndGas(txType string, options objectsMap, networkConfig
 		big.NewInt(0).SetUint64(gasLimit),
 	)
 
-	suggestedFee, gasLimit = adjustTxFeeWithFeeMultiplier(suggestedFee, gasLimit, options)
+	suggestedFee, gasPrice = adjustTxFeeWithFeeMultiplier(suggestedFee, gasPrice, options)
 
 	return suggestedFee, gasPrice, gasLimit, nil
 }
 
-func adjustTxFeeWithFeeMultiplier(txFee *big.Int, gasLimit uint64, options objectsMap) (*big.Int, uint64) {
+func adjustTxFeeWithFeeMultiplier(txFee *big.Int, gasPrice uint64, options objectsMap) (*big.Int, uint64) {
 	feeMultiplierI, ok := options["feeMultiplier"]
 	if !ok {
-		return txFee, gasLimit
+		return txFee, gasPrice
 	}
 
 	feeMultiplier, ok := feeMultiplierI.(float64)
 	if !ok {
-		return txFee, gasLimit
+		return txFee, gasPrice
 	}
 
 	feeMultiplierBig := big.NewFloat(feeMultiplier)
 	bigVal, ok := big.NewFloat(0).SetString(txFee.String())
 	if !ok {
-		return txFee, gasLimit
+		return txFee, gasPrice
 	}
 
 	bigVal.Mul(bigVal, feeMultiplierBig)
 
 	result := new(big.Int)
 	bigVal.Int(result)
 
-	gasLimit = uint64(feeMultiplier * float64(gasLimit))
+	gasPrice = uint64(feeMultiplier * float64(gasPrice))
```

Comment:
you need to check that result is not less than minGasPrice from network.
If it is less than minGasPrice, then return the minGasPrice

Also add a unit test for this scenario.

Reply:
done.